### PR TITLE
Change remember period to range slider

### PR DESCRIPTION
**Pull request by subdavis**
Here's a small change I made to see how #156 might be done.   Haven't bothered styling it yet, but it's totally functional.

Slider all the way left: don't remember
Slider all the way right: remember forever
Anything in the middle: I have this going from 2 to 24 hours, but it could be any range.

screenshot

### CODE REVIEW

**Comment by suBDavis** [comment]
final

I think this one is ready to go - styled the slider to match the unlock button and the general feel of the ui.

**Comment by DukeyToo** [comment]
It would be great if it remembered the original "remember" value before the slider was in place - that would make it a better transition for people already using one of the old button-based remember options.

For example, on first use, my remember slider reverted to "Do not remember" - the actual value was 30 (minutes), which corresponds with the new .5 value.

**Comment by suBDavis** [comment]
Agreed.  Changed internal rememberPeriod value back to using minutes, and updated my new lookup table.  Changes are compatible between master and this feature now.

**Comment by suBDavis** [comment]
I agree.  Reverted back to using minutes for rememberPeriod for compatibility.

File: popups/controllers/masterPasswordController.js [hunk 1]
```diff
@@ -131,7 +166,8 @@ function MasterPasswordController($scope, $routeParams, $location, keepass, unlo
       settings.saveDefaultRememberOptions($scope.rememberPassword, $scope.rememberPeriod);
 
       if ($scope.rememberPeriod) {
-      	settings.setForgetTime('forgetPassword', (Date.now() + (60000*$scope.rememberPeriod)))
+        var check_time = 60000*60*$scope.rememberPeriod; // min per hour * milliseconds per min
```
Comment (DukeyToo) [inline popups/controllers/masterPasswordController.js]: I think this is still using the "hour" unit of measurement, i.e. it is multiplying by an extra 60?

**Comment by subdavis** [comment]
Damn, it's like amateur hour over here.

Fixed and squashed everything down, since some of them were broken and this should be an all-or-nothing change.

**Comment by DukeyToo** [comment]
Thanks much!

**Pull request merged**